### PR TITLE
Fix PHPCS issues, fixes #196

### LINKED ISSUE #196: Fix PHP Code Sniffer Issues
See https://github.com/firegento/firegento-magesetup2/actions/runs/655516803

### General

* Module version: x.x.x
* Magento version: x.x.x
* PHP version: x.x.x

### Issue description

...

### Steps to reproduce

1. ... 
2. ... 
3. ...

### PR DESCRIPTION
**Pull request by sprankhub**
Please make sure these boxes are checked before submitting your PR - thank you!

- [X] Pull request is based against develop branch
- [X] README.md reflects changes (if applicable)
- [X] New files contain a license header

### Issue

This PR fixes issue #196.

### Proposed changes

...

### CODE REVIEW

**Comment by sprankhub** [comment]
@Schrank, @frostblogNet, @mautz-et-tong, could anyone please have a look?

File: .github/workflows/integration.yml [hunk 1]
```diff
@@ -27,9 +24,44 @@ jobs:
...
+            ce_version: '2.4.1-p1'
+      - uses: actions/checkout@v2
+      - name: M2 Integration Tests with Magento 2 Version 2.4.2 (PHP 7.3)
+        uses: extdn/github-actions-m2/magento-integration-tests/7.3@master
+        with:
+            module_name: FireGento_MageSetup
+            composer_name: firegento/magesetup2
+            ce_version: '2.4.2'
+      - uses: actions/checkout@v2
+      - name: M2 Integration Tests with Magento 2 Version 2.4.1-p1 (PHP 7.4)
```
Comment (mautz-et-tong) [inline .github/workflows/integration.yml:55]: duplicate, see line 41

File: Test/Integration/Observer/AddProductAttributeVisibleCheckoutObserverTest.php [hunk 2]
```diff
@@ -20,6 +20,11 @@ public function testExecute()
         $attribute                  = $productAttributeRepository->get('name');
         $this->getRequest()->setParam('attribute_id', $attribute->getAttributeId());
         $this->dispatch($this->uri);
-        self::assertStringContainsString('Visible in Checkout', $this->getResponse()->getBody());
+        // switch to assertStringContainsString completely when we drop 2.3-support
```
Comment (mautz-et-tong) [inline Test/Integration/Observer/AddProductAttributeVisibleCheckoutObserverTest.php]: I would add @ToDo here

**Review by mautz-et-tong — APPROVED** [review]

**Comment by in-session** [comment]
Hi after update into the new merge on Magento 2.4.2 we get the follow error:

```
1 exception(s):
Exception #0 (Magento\Framework\Exception\LocalizedException): Ungültige Methode FireGento\MageSetup\Block\Product\Delivery\Interceptor:: helper

Exception #0 (Magento\Framework\Exception\LocalizedException): Ungültige Methode FireGento\MageSetup\Block\Product\Delivery\Interceptor:: helper
<pre>#1 FireGento\MageSetup\Block\Product\Delivery\Interceptor->__call() called at [vendor/firegento/magesetup2/view/frontend/templates/product/list/delivery.phtml:16]

```
magesetup2/view/frontend/templates/product/list/delivery.phtml:16:
```
//$_helper = $block->helper('Magento\Catalog\Helper\Output');
$_helper = $this->helper(Magento\Catalog\Helper\Output::class);
```

**Pull request merged**